tests/unit/test_plugin.py

```diff
@@ -68,4 +89,25 @@ def test_meter_billing():   # Currently no-op
         timestamp,
         dry_run=True
     )
-    assert status == {}
+    status["tier_1"] == {'status': 'submitted'}
+
+
+@patch('csp_billing_adapter_google.plugin.urllib.request.urlopen')
+def test_meter_billing_fail(mock_urlopen):
+    urlopen = Mock()
+    urlopen.read.side_effect = []
```

Comment:
Not sure what the actual "side effect" is here? Possibly returns None when it tries to pop something from an empty list, or triggers an internal exception because the list is empty...

I would recommend putting an explicit exception in here, or else use `.return_value = None`...

Reply:
This one is really a place holder until we find out what fails actually do. It causes the try block in meter_billing to fall into the exception code.  but I have updated to send back 500 error, which I think is what we were seeing.